src/base/bittorrent/session.h

```diff
@@ -400,6 +407,8 @@ namespace BitTorrent
         void handleTorrentShareLimitChanged(TorrentHandle *const torrent);
         void handleTorrentSavePathChanged(TorrentHandle *const torrent);
         void handleTorrentCategoryChanged(TorrentHandle *const torrent, const QString &oldCategory);
+        void handleTorrentTagAdded(const QString &tag, int new_tag_count);
```

Comment:
As I said above it should be changed to `void handleTorrentTagAdded(TorrentHandle *const torrent, const QString &tag)`.